From: Dylan Maxwell <maxwelld@frib.msu.edu>
Date: Thu, 12 May 2016 10:11:41 -0700
Subject: Replace static URL with constructor parameter.

---
 channelfinder/ChannelFinderClient.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/channelfinder/ChannelFinderClient.py b/channelfinder/ChannelFinderClient.py
index 88adfdd..6e44b6a 100644
--- a/channelfinder/ChannelFinderClient.py
+++ b/channelfinder/ChannelFinderClient.py
@@ -49,7 +49,7 @@ class ChannelFinderClient(object):
             else:
                 self.__auth = None
             self.__session = requests.Session()
-            self.__session.mount('https://localhost:8181/ChannelFinder/', Ssl3HttpAdapter())
+            self.__session.mount(BaseURL, Ssl3HttpAdapter())
             self.__session.get(self.__baseURL, verify=False, headers=copy(self.__jsonheader)).raise_for_status()
  
         except:
