From: Dylan Maxwell <maxwelld@frib.msu.edu>
Date: Thu, 12 May 2016 10:14:08 -0700
Subject: Replace obsolete SSLv3 protocol with recommended SSLv23.

---
 channelfinder/ChannelFinderClient.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/channelfinder/ChannelFinderClient.py b/channelfinder/ChannelFinderClient.py
index 740bf14..672fa09 100644
--- a/channelfinder/ChannelFinderClient.py
+++ b/channelfinder/ChannelFinderClient.py
@@ -614,6 +614,6 @@ class Ssl3HttpAdapter(HTTPAdapter):
         self.poolmanager = PoolManager(num_pools=connections,
                                        maxsize=maxsize,
                                        block=block,
-                                       ssl_version=ssl.PROTOCOL_SSLv3)
+                                       ssl_version=ssl.PROTOCOL_SSLv23)
 
         
